apps/condo/domains/notification/schema/SendMessageService.js

```diff
@@ -143,6 +144,22 @@ const SendMessageService = new GQLCustomSchema('SendMessageService', {
                 if (to.user) messageAttrs.user = { connect: to.user }
                 if (to.remoteClient) messageAttrs.remoteClient = { connect: to.remoteClient }
 
+                // Trying to call hook to define should the message be created or not
+                /** @constructor */
+                const hooksClass = get(MESSAGE_META, [type, 'hooksClass'])
```

Comment:
We have ADR-7 settings for messages that handle transport selection for each message type deep inside deliverMessage transport adapters. This way it conflicts with current logic, because any message type could have any set of transports which could be changed any time. If such case happens, this logic will break functionality, for ex. by blocking sending of sms/emails messages

Reply:
Fixed. All logic was moved to the corresponding task.